setup/system.sh

```diff
@@ -144,6 +144,10 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Launching SSH public key creation...'
+
+nohup ssh-keygen -t rsa -b 2048 -a 100 -f /root/.ssh/id_rsa_miab -N '' -q 2>&1 >/dev/null
```

Comment:
well, originally when I wrote this I was building a much stronger key (`-b 8192`) which takes more than a few seconds to generate, so that's right to do that in a non `nohup` with a weaker key.